musl: use fileno() instead of the glibc FILE internal _fileno.

diff --git a/main/exteplayer.c b/main/exteplayer.c
index 062c16f..91c512d 100755
--- a/main/exteplayer.c
+++ b/main/exteplayer.c
@@ -332,8 +332,8 @@ static void SetBuffering()
     }
     
     // make fgets not blocking 
-    int flags = fcntl(stdin->_fileno, F_GETFL, 0); 
-    fcntl(stdin->_fileno, F_SETFL, flags | O_NONBLOCK); 
+    int flags = fcntl(fileno(stdin), F_GETFL, 0); 
+    fcntl(fileno(stdin), F_SETFL, flags | O_NONBLOCK); 
 }
 
 static void SetNice(int prio)
